device-types/Extreme Networks/X450-G2-48p-10GE4.yaml

```diff
@@ -0,0 +1,228 @@
+---
+manufacturer: Extreme Networks
+model: X450-G2-48p-10GE4
+slug: extreme-networks-x450-g2-48p-10ge4
+part_number: '16179'
+u_height: 1
+is_full_depth: true
+weight: 6450
+comments: '[X450-G2-48p-10GE4 Datasheet](https://cloud.kapostcontent.net/pub/6f346f0e-30e3-452c-86cd-56795f6a1a65/x450-g2-data-sheet.pdf?kui=bKqLVAj7ueLS6LTgIp2rvw)'
+weight_unit: g
+airflow: front-to-rear
+subdevice_role: parent
```

Comment:
This is generally used for chassis which house other devices, this is not meant for devices with modules. Is this actually needed, and if so why are there no `device-bays`?